### ISSUE TITLE
ARM architecture support

### ISSUE THREAD

**Issue by amineism** [hadriansecurity/sanicdns#14]
Hello,

Interested in trying the tool and wanted to know if this was currently only compatible with AMD64 architectures. If so, is there any plan to make it ARM compatible ? Thanks

Currently trying to make it work on ARM but some parts of the code seem to be using x86-specific intrinsic functions. Getting the following errors when running `ninja -C build`. Also had to remove the `-mssse3` and `-msse4.2` flags from the `build.ninja` file for attain this step in ARM. Had to make a few changes in the `apt install` command as well.

**Comment by kalmjasper** [comment]
Yes I'd like to rewrite this code to enable the compiler to insert the ARM/x86 vector instructions. Coming up :)

**Comment by kalmjasper** [comment]
You can try it out with any random x86 VPS for now. Adding ARM support is definitely a must nowadays though

**Comment by Azathothas** [comment]
Hi, 
now that native arm runners are available on GitHub, I hope you add releases for this architecture as well.
Thanks for making this awesome tool

**Comment by kalmjasper** [comment]
Hey there! @Geertiebear Is currently quite busy, we'll schedule a day to make the ARM functionality work when his schedule allows for it. Good to hear you're using SanicDNS :)
